src/background.ts

```diff
@@ -14,7 +14,7 @@ import { EditorCmds } from "@src/background/editor"
 import * as convert from "@src/lib/convert"
 import * as config from "@src/lib/config"
 import * as dom from "@src/lib/dom"
-import * as download_background from "@src/background/download_background"
+import * as download_background from "@src/background_lib/download_background"
```

Comment:
Consider just removing this import and its use?

Unclear why you're importing the real library rather than its wrapper.